lib/tasks/after_tasks.rake

```diff
@@ -249,7 +249,7 @@ namespace :After do
 #  task(:invite_external_authors => :environment) do
 #    Invitation.where("sent_at is NULL").where("external_author_id IS NOT NULL").each do |invite|
 #      archivist = invite.external_author.external_creatorships.collect(&:archivist).collect(&:login).uniq.join(", ")
-#      UserMailer.invitation_to_claim(invite, archivist).deliver
+#      UserMailer.invitation_to_claim(invite, archivist).deliver_now
```

Comment:
I modeled it after the other `invitation_to_claim` mail: https://github.com/otwcode/otwarchive/blob/e929271f4ced8fd8858e1ffd55128ca44ba079dc/app/models/invitation.rb#L70-L71
I suspect the commented-out code won't work regardless, because it's passing the invite instead of its ID as the first argument, but I figure that's not really in scope.